### ISSUE TITLE
bad cbst result

### ISSUE THREAD

**Issue by Luffy03** [Junjue-Wang/LoveDA#8]
hello, we re-run the cbst_train with the default settings you provide, but get bad results as shown in the fig, even worse than the source only method. i wonder the stability of the training of cbst, and i will appreciate that if you can provide the training log of the cbst. THANK YOU VERY MUCH!
![Uploading 屏幕截图 2021-11-11 112454.png…]()

**Comment by Luffy03** [comment]
[scores.txt](https://github.com/Junjue-Wang/LoveDA/files/7517546/scores.txt)

**Comment by Junjue-Wang** [comment]
This is the result of 2Urban experiments or 2Rural experiments?

**Comment by Luffy03** [comment]
> This is the result of 2Urban experiments or 2Rural experiments?

2urban

**Comment by Junjue-Wang** [comment]
This is our reported log.
[2urban_cbst.txt](https://github.com/Junjue-Wang/LoveDA/files/7518045/2urban_cbst.txt)

**Comment by Luffy03** [comment]
> This is our reported log. [2urban_cbst.txt](https://github.com/Junjue-Wang/LoveDA/files/7518045/2urban_cbst.txt)

First, thanks for your kind answering!
But I have assumed that the miou 0.4132 is for the test dataset, but after reading the log, it seems that the miou 0.4132 is for the val urban dataset? 
![屏幕截图 2021-11-11 135949](https://user-images.githubusercontent.com/52514670/141247671-9730b6eb-0abe-4284-8e3a-a19bc10615e7.png)

**Comment by Luffy03** [comment]
We further run the eval.py with your pre-trained model on the urban eval dataset, the results gets only 0.32399. Is it any mistake with the provided pre-trained model?
[1636610453.7992563.log](https://github.com/Junjue-Wang/LoveDA/files/7518346/1636610453.7992563.log)

**Comment by Junjue-Wang** [comment]
> We further run the eval.py with your pre-trained model on the urban eval dataset, the results gets only 0.32399. Is it any mistake with the provided pre-trained model? [1636610453.7992563.log](https://github.com/Junjue-Wang/LoveDA/files/7518346/1636610453.7992563.log)

This was reported on our test set. You can use our pre-trained weights for predictions and submit them on our website. Please let me if you have reproduced the CBST results on the Codalab with the pre-trained weights. Thank you.

**Comment by Luffy03** [comment]
i evaluated the pre-trained 'CBST_2Urban.pth' on the web just now.
[scores.txt](https://github.com/Junjue-Wang/LoveDA/files/7518449/scores.txt)
it seems something wrong? we have not changed any code
sorry for bothering again and again

**Comment by Luffy03** [comment]
What is more, the results reported in the paper is for the val dataset or test dataset?
![141247671-9730b6eb-0abe-4284-8e3a-a19bc10615e7](https://user-images.githubusercontent.com/52514670/141253592-ee42bf3e-e4ae-43cb-b7df-2f734c182f42.png)
![2021-11-11 15-07-14屏幕截图](https://user-images.githubusercontent.com/52514670/141253621-746af758-394a-4f2b-bab7-2fbec1fbc17e.png)

**Comment by Junjue-Wang** [comment]
For test dataset. We have re-evaluated based on our weights and reproduced the results 41.32%.
We think we may upload the wrong weights. And the right weights will be updated later.
![cbst](https://user-images.githubusercontent.com/27996790/141257826-9ddfd0c1-9aaf-49b8-8c89-d8152e86c5d1.png)

**Comment by Luffy03** [comment]
Thank you very much!

**Comment by Junjue-Wang** [comment]
I have already updated the pre-weights and added the predict.py

**Comment by Luffy03** [comment]
> I have already updated the pre-weights and added the predict.py

thanks！

**Comment by Junjue-Wang** [comment]
You are welcome. Please feel free if you have any further questions.
